cloudify_secrets/tasks.py

```diff
@@ -0,0 +1,92 @@
+########
+# Copyright (c) 2019 GigaSpaces Technologies Ltd. All rights reserved
```

Comment:
Replace `GigaSpaces Technologies` to `Cloudify Platform Ltd.`.

Reply:
done